chrome/content/exteditor.js

```diff
@@ -189,7 +189,7 @@ function updateEditor() {
                 if (prefEditHeaders[exteditor_REPLY_TO]) msgCompFields.replyTo = headerHash[exteditor_REPLY_TO.toLowerCase()];
                 if (prefEditHeaders[exteditor_NEWSGROUP]) msgCompFields.newsgroups = headerHash[exteditor_NEWSGROUP.toLowerCase()];
 
-                CompFields2Recipients(msgCompFields);
+                extEditorCompFields2Recipients(msgCompFields);
```

Comment:
I could keep the call to the broken version of `CompFields2Recipients` and add compatibility stuff around that, but I feared that `CompFields2Recipients` will change behavior again or even be removed since the function seems like dead code to me in the current comm-central.